crates/nu-command/src/random/chars.rs

```diff
@@ -31,6 +31,17 @@ impl Command for SubCommand {
         "Generate random chars"
     }
 
+    fn search_terms(&self) -> Vec<&str> {
+        vec![
+            "random",
+            "rand",
+            "generate",
+            "char",
+            "symbol",
+            "alphanumeric",
```

Comment:
As an example, for those terms we should already find the command. (There is a test that will warn about duplicated names.)

```suggestion
            "generate",
            "symbol",
            "alphanumeric",
```